sys-devel/llvm/patches/clang-6.0.1.patchset

```diff
@@ -170,5 +170,53 @@ index 8b5b48e..ccd851f 100644
    std::string findLibCxxIncludePath() const override;
    void addLibStdCxxIncludePaths(
 -- 
-2.16.4
+2.19.1
+
+
+From b37aa9a5bd0be6b593eb0926195ca23a6e1b3629 Mon Sep 17 00:00:00 2001
+From: Calvin Hill <calvin@hakobaito.co.uk>
+Date: Sat, 20 Oct 2018 22:44:49 +0100
+Subject: clang: Enable float128 support in clang with tests
+
+
+diff --git a/lib/Basic/Targets/OSTargets.h b/lib/Basic/Targets/OSTargets.h
+index 790732b..9f049bf 100644
+--- a/lib/Basic/Targets/OSTargets.h
++++ b/lib/Basic/Targets/OSTargets.h
+@@ -251,6 +251,10 @@ protected:
+     Builder.defineMacro("__HAIKU__");
+     Builder.defineMacro("__ELF__");
+     DefineStd(Builder, "unix", Opts);
++    if (this->HasFloat128) {
++      Builder.defineMacro("__FLOAT128__");
++      Builder.defineMacro("_GLIBCXX_USE_FLOAT128");
++    }
+   }
+ 
+ public:
+@@ -260,6 +264,7 @@ public:
+     this->IntPtrType = TargetInfo::SignedLong;
+     this->PtrDiffType = TargetInfo::SignedLong;
+     this->ProcessIDType = TargetInfo::SignedLong;
++    this->HasFloat128 = true;
```

Comment:
I would use the same code as Solaris:
https://github.com/llvm-mirror/clang/commit/41ef6cd00386db07cc02e3b618873e87527170da#diff-ae1b30e40847dc1832f14faa7dc8d174R563

Reply:
Patchset updated.